esp/services/ws_logaccess/WsLogAccessService.cpp

```diff
@@ -13,11 +13,85 @@ bool Cws_logaccessEx::onGetLogAccessInfo(IEspContext &context, IEspGetLogAccessI
     bool success = true;
     if (queryRemoteLogAccessor())
     {
+        IPropertyTree * logColumnDefinitions = queryRemoteLogAccessor()->queryLogMap();
+        if (!logColumnDefinitions)
+        {
+            success = false;
+            OERRLOG("WsLogaccess: NULL col defintions - check logaccess logmap config!");
+        }
+        else
+        {
+            double clientVer = context.getClientVersion();
+
+            if (clientVer >= 1.04)
+            {
+                IArrayOf<IEspLogColumn> logColumns;
+                Owned<IPropertyTreeIterator> columnIter = logColumnDefinitions->getElements("logMaps");
+                ForEach(*columnIter)
+                {
+                    IPropertyTree & column = columnIter->query();
+                    if (column.hasProp("@searchColumn"))
+                    {
+                        Owned<IEspLogColumn> espLogColumn = createLogColumn();
+
+                        const char * csearchColumn=column.queryProp("@searchColumn");
+                        const char * cprojectName=column.queryProp("@projectName");
+                        espLogColumn->setName(!isEmptyString(cprojectName) ? cprojectName: csearchColumn);
+
+                        if (column.hasProp("@type"))
+                            espLogColumn->setLogType(column.queryProp("@type"));
+
+                        if (column.hasProp("@columnMode"))
+                            espLogColumn->setColumnMode(column.queryProp("@columnMode"));
+
+                        if (column.hasProp("EnumValues"))
+                        {
+                            Owned<IPropertyTreeIterator> enumValues = column.getElements("EnumValues");
+                            StringArray enumValuesArr;
+                            ForEach(*enumValues)
+                            {
+                                IPropertyTree & enumVal = enumValues->query();
```

Comment:
trivial: to be consistent with line 11, remove the space from the front of the '*'.